components/event-bus/internal/knative/subscription/controller/eventactivation/reconciler.go

```diff
@@ -59,11 +59,11 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	// updates regardless of whether the reconcile error out.
 	requeue, reconcileErr := r.reconcile(ctx, ea)
 	if reconcileErr != nil {
-		log.Error(reconcileErr, "Reconciling EventActivation")
+		log.Error(reconcileErr, "Error reconciling EventActivation")
```

Comment:
Log has "ERROR" as a prefix, so that it's not necessary to put it in the text message

Reply:
Done